fares-data-build-tool/fdbt-site
# Feature/tfn 462 add user type info matching json

# Description

-   To add the passenger/user type info to the matching JSON that gets written to S3 for all of our exiting user journeys.

# Testing instructions

-   The branch can be tested locally using the fdbt-dev repo. The matching JSON that gets written locally to S3 can be checked to ensure that the expected data is present and in the right format. I have done this for the 'Any' passenger type and saw that the passenger type as a property was correctly written to S3 in the JSON. NOTE: Once @dannydaviesIW has finished his ticket (453) we will need to check that the data that we are potentially capturing for other passenger types (i.e. min age, max age & document proof) also gets written correctly to S3 in the JSON.

# Type of change


# Checklist:
